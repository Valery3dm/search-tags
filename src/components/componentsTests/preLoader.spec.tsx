import Enzyme, { shallow } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import toJson from 'enzyme-to-json';

import { PreLoader } from '../index';

Enzyme.configure({ adapter: new Adapter() });

describe('PreLoader component:', () => {
    
    it('PreLoader Snapshot component', () => {
        const component = shallow(<PreLoader />);
        expect(toJson(component)).toMatchSnapshot();
    });

    it('PreLoader TAG', () => {
        const component = shallow(<PreLoader />);
        expect(component.find('div div').text()).toEqual('Please enter TAG');
    });
});
