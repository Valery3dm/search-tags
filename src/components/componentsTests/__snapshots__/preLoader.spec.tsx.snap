// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreLoader component: PreLoader Snapshot component 1`] = `
<div>
  <styled.div>
    &lt; ðŸŒ¸ &gt;
  </styled.div>
  <div>
    Please enter TAG
  </div>
</div>
`;
